dev/infra/Dockerfile

```diff
@@ -92,7 +104,7 @@ RUN mkdir -p /usr/local/pypy/pypy3.8 && \
     ln -sf /usr/local/pypy/pypy3.8/bin/pypy /usr/local/bin/pypy3.8 && \
     ln -sf /usr/local/pypy/pypy3.8/bin/pypy /usr/local/bin/pypy3
 RUN curl -sS https://bootstrap.pypa.io/get-pip.py | pypy3
-RUN pypy3 -m pip install numpy 'six==1.16.0' 'pandas<=2.1.4' scipy coverage matplotlib lxml
+RUN pypy3 -m pip install numpy pyarrow 'six==1.16.0' 'pandas==2.1.4' scipy coverage matplotlib lxml
```

Comment:
If it works, let's go ahead. LGTM

Reply:
Okay, `pyarrow` can be installed successfully. Currently, testing has found that `only one UT` cannot pass. Let me try:
1. Install pyarrow
2. Ignoring this UT
Let me test it.



Let me test it